main.py

```diff
@@ -0,0 +1,48 @@
+FILE_NAME = 'items.txt'
+MAX_WEIGHT_DAG = 400
+
+
+class Pack(object):
+    def __init__(self, value=None, weight=None, file_name=FILE_NAME):
+        self.value = value
+        self.weight = weight
+        self.file_name = file_name
+        self.text_list = self.read_file(self.file_name)
+
+    @staticmethod
+    def read_file(file_name):
+        return [line.strip().split(',') for line in open(file_name)]
+
+    def convert_to_obj(self):
+        return [type(line[0], (object,),
+                     {'value': int(line[1]), 'weight': int(line[2])})
```

Comment:
` {'weight': int(line[1]), 'value': int(line[2])})` value and weight are mixed

Reply:
oh, yes. thank you